# HG changeset patch
# User Rik <rik@octave.org>
# Date 1372016993 25200
#      Sun Jun 23 12:49:53 2013 -0700
# Node ID a4969508008ef7639f463f76c4b98717c870d4f5
# Parent  71fc195d3e84a62df424468754d75e023f922a72
doc: Periodic spellcheck of the documentation.

* doc/interpreter/doccheck/aspell-octave.en.pws: Update Octave-specific
Aspell dictionary.

* doc/interpreter/contrib.txi, doc/interpreter/errors.txi,
doc/interpreter/expr.txi, doc/interpreter/io.txi, doc/interpreter/preface.txi,
doc/interpreter/sparse.txi, doc/interpreter/stmt.txi,
doc/interpreter/strings.txi, libinterp/corefcn/qz.cc,
libinterp/dldfcn/ccolamd.cc, libinterp/dldfcn/fftw.cc,
libinterp/interpfcn/graphics.cc, scripts/general/structfun.m,
scripts/image/ind2rgb.m, scripts/image/rgb2ind.m, scripts/io/importdata.m,
scripts/linear-algebra/duplication_matrix.m, scripts/linear-algebra/housh.m,
scripts/miscellaneous/mkoctfile.m, scripts/plot/comet3.m, scripts/plot/gco.m,
scripts/plot/legend.m, scripts/plot/print.m, scripts/signal/fftshift.m,
scripts/signal/periodogram.m, scripts/sparse/pcg.m,
scripts/statistics/tests/f_test_regression.m,
scripts/statistics/tests/t_test_regression.m, scripts/strings/strjoin.m,
scripts/strings/strsplit.m: Correct spellings or add @nospell macro.

diff --git a/doc/interpreter/contrib.txi b/doc/interpreter/contrib.txi
--- a/doc/interpreter/contrib.txi
+++ b/doc/interpreter/contrib.txi
@@ -85,18 +85,18 @@ hg export -o ../cool.diff tip
                              # export the changeset to a diff
                              # file
 # attach ../cool.diff to your bug report
 @end group
 @end example
 
 You may want to get familiar with Mercurial queues to manage your
 changesets.  For working with queues you have to activate the extension
-mq with the following entry in Mercurials config file @file{.hgrc} (or
-@file{Mercurial.ini} on Windows):
+@nospell{mq} with the following entry in Mercurial's configuration file
+@file{.hgrc} (or @file{Mercurial.ini} on Windows):
 
 @example
 @group
 [extensions]
 mq=
 @end group
 @end example
 
@@ -144,17 +144,17 @@ The following entries in the @file{.hgrc
 [extensions]
 graphlog=
 color=
 progress=
 pager=
 @end group
 @end example
 
-For the colour extension, default colour and formatting
+For the color extension, default color and formatting
 of @code{hg status} can be modified by
 
 @example
 @group
 [color]
 status.modified = magenta bold
 status.added = green bold
 status.removed = red bold
@@ -177,21 +177,20 @@ pager = LESS='FSRX' less
 # Some commands that aren't paged by default; also enable paging
 # for them
 attend = tags, help, annotate, cat, diff, export, status, \
          outgoing, incoming
 @end group
 @end example
 
 Enabling the described extensions should immediately lead to a difference
-when using the command line version of hg.  Of these options, the only one
-that enables a new command is graphlog.  It is recommanded that you use the
-command @code{hg glog} instead of @code{hg log} for a better feel
-what commits are being based on.
-
+when using the command line version of @nospell{hg}.  Of these options, the
+only one that enables a new command is @nospell{graphlog}.  It is recommanded
+that you use the command @code{hg glog} instead of @code{hg log} for a better
+feel what commits are being based on.
 
 @node General Guidelines
 @section General Guidelines
 
 All Octave's sources are distributed under the General Public License
 (GPL).  Currently, Octave uses GPL version 3.  For details about this
 license, see @url{http://www.gnu.org/licenses/gpl.html}.  Therefore,
 whenever you create a new source file, it should have the following
diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -15,16 +15,17 @@ Alamos
 aleph
 alnum
 alphanum
 amd
 ANOVA
 anova
 Anymap
 API
+appdata
 approximant
 arg
 ARMA
 arpack
 ascii
 Ashok
 async
 atan
@@ -44,16 +45,17 @@ autoregressions
 autoscaled
 AutoScaling
 AWK
 awt
 backend
 Backends
 backends
 bartlett
+BaseValue
 Bateman
 BDF
 bdf
 betacdf
 betainv
 betaln
 betapdf
 betarnd
@@ -77,63 +79,65 @@ bof
 boolean
 boolMatrix
 boxerrorbars
 boxoff
 boxon
 boxxyerrorbars
 bracketx
 brackety
+braindead
 breakpoint
 Brenan
 broadcastable
 Brockwell
 BSX
 builtin
 builtins
 ButtonDownFcn
 BV
 bz
 BZ
 bzip
 CallbackObject
 callee's
 camd
+CancelString
 cauchy
 caxis
 ccolamd
-CCR
-CCS
 cd
 cdata
 CDF
 cdf
 CDFs
 cdot
 ceil
 cellstr
 ChangeLog
 changelogs
 changeset
 changesets
 Chaves
 chdir
+Chebyshev
 chisq
 chisquare
 chol
 Cholesky
 cholmod
 chrominance
 cindex
 circ
 circshift
 circum
 classpath
 classpaths
 Clenshaw
+CLI
 clim
 cloglog
 clubsuit
 CMatrix
 cmd
 cmember
 cntrl
 cof
@@ -171,58 +175,58 @@ csymamd
 ctranspose
 CTRL
 CTS
 cummax
 cummin
 cumprod
 cumsum
 cURL
+CurrentObject
 Cuthill
 cxsparse
 Cygwin
 DAE
 DAEs
 daspk
 dasrt
 dassl
 dataset
 datasets
 datasource
 datestr
 datestrings
 davis
 ddd
 dddd
-dddddddddd
 deallocate
 deallocated
 deconv
 deftypefn
 Delaunay
 delaunay
 delaunayn
 DeleteFcn
 delim
 deltaX
 demi
 Demmel
 DeskJet
-Deskjet
 det
 diag
 diamondsuit
 dir
 disp
+DisplayName
+DisplayNames
 displayrange
 dMatrix
 dmperm
-DMS
-Dn
 Dobkin
+docstrings
 dOmega
 dotall
 dotexceptnewline
 Downarrow
 downarrow
 downdate
 dpi
 Driebeck
@@ -255,16 +259,17 @@ EOF
 EOLs
 EOS
 eps
 eq
 equidistributed
 equispaced
 erf
 erfc
+erfi
 errno
 Errorbar
 errorbar
 Errorbars
 errorbars
 errordlg
 ErrorHandler
 Esmond
@@ -285,17 +290,17 @@ fcdf
 femmodel
 FFF
 FFT
 fft
 fftn
 fftpack
 FFTs
 fftw
-FG
+Fiedler
 fieldname
 fieldnames
 filename
 filenames
 Filesystem
 FinDiffType
 finv
 FIRfilter
@@ -304,16 +309,17 @@ FlashPix
 FLTK
 fltk
 fminunc
 fontconfig
 fontname
 forall
 foregroundcolor
 formfeed
+Forsythe
 Fortran
 fpdf
 fprintf
 FreeBSD
 freespacing
 FreeType
 freetype
 frnd
@@ -332,17 +338,16 @@ GCC
 gcd
 ge
 GECOS
 genvarname
 geocdf
 geoinv
 geopdf
 geornd
-GEP
 geq
 gesdd
 gesvd
 gfortran
 Ghostscript
 Ghostscript's
 gif
 GIF
@@ -389,16 +394,17 @@ hdf
 headerlines
 heartsuit
 helpdlg
 Helvetica
 Hermitian
 Hessenberg
 heteroscedascity
 hggroup
+hggroups
 hh
 HH
 Higham
 Hindmarsh
 Hindmarsh's
 Hodel
 holomorphic
 Horner's
@@ -412,68 +418,71 @@ Hudak
 Huhdanpaa
 hygecdf
 hygeinv
 hygepdf
 hygernd
 Hyndman
 Hypergeometric
 hypergeometric
+IEC
 IEEE
 ifelse
-iff
 ifft
 ifftn
 ignorecase
-IIP
 ij
 Im
 imag
 ImageMagick
 Imenu
 Indices
 indices
 infty
 init
+InitialValue
 inline
 Inline
 inpolygon
 inputdlg
 internet
 interp
 interpderiv
 interpft
 interpn
 interpolant
 Interpolants
 interquartile
 inv
+involutory
 ipermute
 ishandle
 ishghandle
 isolines
 isosurface
 isovalue
 isreal
 issparse
 isvector
 iter
 ith
 iy
+iz
 Jacobian
 Jacobians
 javaaddpath
 javamem
-ji
 JIT
 jpeg
 JPEG
 jpg
 jvm
 JVM's
+Kac
+Kahan
 keybindings
 keypress
 Kolmogorov
 kolmogorov
 Konrod
 Krishnamurthy
 kron
 kruskal
@@ -482,25 +491,27 @@ Krylov
 kurtosis
 LabelSpacing
 langle
 Lanzcos
 lapack
 laplace
 Larimore
 LaserJet
+Lauchli
 lceil
 ldiv
 ldivide
 ldots
 le
 Leftarrow
 leftarrow
 Leftrightarrow
 leftrightarrow
+Lehmer
 Lehoucq
 leq
 Levinson
 LF
 lfloor
 li
 libcurl
 liblapack
@@ -510,16 +521,18 @@ Libtool
 licensors
 lineanchors
 linefeeds
 linesearch
 linespec
 linespoints
 linkprop
 listdlg
+ListSize
+ListString
 literalspacing
 Liu
 LLVM
 LM
 lm
 loadpath
 Lobatto
 logit
@@ -527,16 +540,17 @@ logncdf
 logninv
 lognormal
 lognpdf
 lognrnd
 lookup
 Lookup
 lookups
 Los
+Lotkin
 Lovato
 lpx
 lr
 lsode
 lt
 lu
 Lua
 luinc
@@ -589,17 +603,16 @@ mkdir
 mkoctfile
 mldivide
 mmd
 MMM
 mmm
 mmmm
 mmmyy
 mmmyyyy
-modelled
 Moler
 Montanet
 mpoles
 mpower
 mput
 mrdivide
 msgbox
 MSYS
@@ -609,16 +622,18 @@ multi
 multibyte
 multiline
 multipledelimsasone
 MultiSelect
 mxArray
 myclass
 myfun
 nabla
+namespace
+NAMESPACE
 NaN
 NaNs
 nargin
 nbincdf
 nbininv
 nbinpdf
 nbinrnd
 nd
@@ -633,40 +648,42 @@ nfev
 nfft
 Ng
 ni
 Nishimura
 NLP
 nnls
 nnz
 nocompute
+nodither
 nolabel
 noncommercially
 nonconformant
 nonsmooth
 nonzeros
+NOP
 noperm
 normcdf
 normest
 norminv
 normpdf
 normrnd
 NorthOutside
 noscal
 noshare
 notin
 nthargout
 NTSC
-nul
+ntsc
 Numpy
-Nx
 nzmax
 oct
 octaverc
 ODEPACK
+OKString
 OLS
 onCleanup
 online
 OpenGL
 oplus
 Oppenheim
 oregonator
 Orthogonalize
@@ -675,29 +692,31 @@ otimes
 OutputFcn
 outputfcn
 overridable
 paperorientation
 paperposition
 papersize
 paperunits
 parseparams
+Parter
 pbm
 PBM
 PBMplus
 pcg
 PCG
 pchip
 pcr
 pcre
 PCRE
 PCX
 pcx
 pdf
 PDF
+pentadiagonal
 periodogram
 perp
 Petzold's
 PGF
 pgm
 PGMRES
 PHP
 pict
@@ -714,38 +733,40 @@ poissrnd
 polyderiv
 polyeig
 polyfit
 polyval
 POSIX
 postorder
 PostScript
 Pothen
+powerset
 pre
 preconditioner
 premultiplied
 Prepend
 prepended
 presolver
 printf
 priori
 Profiler
 profiler
+prolate
+PromptString
 propto
 proven
 ps
 pseudoinverse
 pseudonorm
 pseudorandom
 Pseudospectra
+pseudospectrum
 ptrace
 punct
-PWS
 Pxx
-Qci
 Qhull
 qhull
 QP
 QQ
 qrupdate
 QRUPDATE
 QScintilla
 quadcc
@@ -768,64 +789,64 @@ rangle
 ranlib
 ras
 Rawlings
 rceil
 rdivide
 Readline
 readline
 recursing
+Redheffer
 reentrant
 regex
 regressor
-reimported
 Reindent
 relicensing
 ren
 renderer
 repelems
 repmat
 resampled
 resampling
 Resize
 resize
 resized
 resnorm
 resparsify
 RET
 returnonerror
 rfloor
+rgb
 RGB
 Riccati
 Rightarrow
 rightarrow
 rline
 rmdir
 RMS
 rnd
 roffset
 Rosser
 Rossum
 rpath
 RPMs
-rr
 runtime
 sa
 Saad
 Sandia
 SAS
-sB
 Schafer
 schar
 Schur
 SCO
 screenful
 se
 sed
 seealso
+SelectionMode
 semidefinite
 Sep
 Shampine
 shiftdim
 si
 signum
 SIGNUM
 sim
@@ -881,16 +902,17 @@ STFT
 STL
 stochastically
 strcmp
 stringanchors
 strncmp
 strncmpi
 strread
 strread's
+strsplit
 struct
 structs
 subarrays
 subdirectories
 subdirectory
 subexpressions
 subfunction
 Subfunction
@@ -931,24 +953,26 @@ SVG
 svg
 SwitzerlandLight
 Sym
 sym
 SYMAMD
 symamd
 symbfact
 symrcm
+Szego
 tcdf
 Tcv
 terminal's
 tex
 texinfo
 Texinfo
 TextAlphaBits
 textfield
+textread
 textscan
 th
 ths
 tif
 Tikhonov
 TikZ
 tinv
 Tisseur
@@ -967,33 +991,33 @@ traceback
 trapz
 treatasempty
 treelayout
 treeplot
 tridiagonal
 triplot
 trnd
 Tsang
+Tukey
 tuples
 Tx
 txi
 typedefs
 TypicalX
 ub
 UB
 uchar
 UID
 uimenu
 uint
 uiputfile
 ulong
 Ultrix
 umfpack
 uminus
-un
 Unary
 unary
 uncomment
 Uncomments
 unformatted
 unidcdf
 unidinv
 unidpdf
@@ -1004,16 +1028,17 @@ UniformOutput
 unifpdf
 unifrnd
 unimodal
 Uninstall
 uninstalled
 univariate
 unnormalized
 unpadded
+unpermuted
 unpivoted
 unregister
 unshare
 unsymmetric
 untabified
 uparrow
 Uparrow
 uplus
@@ -1033,21 +1058,22 @@ vectorize
 vectorizing
 vee
 versa
 vertcat
 vertices
 Villadsen
 voronoi
 Voronoi
-Wa
 waitbar
 waitbars
 wallis
 warndlg
+Wathen
+WAV
 wblcdf
 wblinv
 wblpdf
 wblrnd
 WebKit
 Weibull
 Welch
 welch
@@ -1061,17 +1087,16 @@ wilcoxon
 Wilcoxon
 wildcard
 Wildcards
 wildcards
 Wilks
 windowstyle
 WinRand
 WIPO
-Wl
 wp
 wspace
 xb
 xc
 xcorr
 xdigit
 xerrorbar
 xerrorbars
@@ -1080,16 +1105,17 @@ xGBTRF
 xGELSD
 xGETRF
 xGTSV
 xl
 xmin
 xPBTRF
 xPOTRF
 xPTSV
+xtest
 xTRTRS
 xu
 xwd
 xy
 xyerrorbar
 xyerrorbars
 xyz
 yc
@@ -1099,12 +1125,11 @@ yerrorbars
 ylim
 yy
 YY
 yyyy
 YYYY
 yyyymmdd
 yyyymmddTHHMMSS
 Zechner
-zer
 Ziggurat
 zlib
 zlim
diff --git a/doc/interpreter/errors.txi b/doc/interpreter/errors.txi
--- a/doc/interpreter/errors.txi
+++ b/doc/interpreter/errors.txi
@@ -184,17 +184,17 @@ It is also possible to assign an identif
 If an error has such an ID the user can catch this error
 as will be shown in the next example.  To assign an ID to an error,
 simply call @code{error} with two string arguments, where the first
 is the identification string, and the second is the actual error.  Note
 that error IDs are in the format "NAMESPACE:ERROR-NAME".  The namespace
 "Octave" is used for Octave's own errors.  Any other string is available
 as a namespace for user's own errors.
 
-The next example counts indexing errors.  The errors are catched using the
+The next example counts indexing errors.  The errors are caught using the
 field identifier of the structure returned by the function @code{lasterror}.
 
 @example
 @group
 number_of_errors = 0;
 for n = 1:100
   try
     @dots{}
diff --git a/doc/interpreter/expr.txi b/doc/interpreter/expr.txi
--- a/doc/interpreter/expr.txi
+++ b/doc/interpreter/expr.txi
@@ -763,17 +763,17 @@ True if @var{x} is greater than @var{y}.
 @itemx @var{x} ~= @var{y}
 @opindex !=
 @opindex ~=
 True if @var{x} is not equal to @var{y}.
 @end table
 
 For complex numbers, the following ordering is defined:
 @var{z1} < @var{z2}
-iff
+if and only if
 
 @example
 @group
   abs (@var{z1}) < abs (@var{z2}) 
   || (abs (@var{z1}) == abs (@var{z2}) && arg (@var{z1}) < arg (@var{z2}))
 @end group
 @end example
 
diff --git a/doc/interpreter/io.txi b/doc/interpreter/io.txi
--- a/doc/interpreter/io.txi
+++ b/doc/interpreter/io.txi
@@ -17,17 +17,17 @@
 @c <http://www.gnu.org/licenses/>.
 
 @node Input and Output
 @chapter Input and Output
 
 Octave supports several ways of reading and writing data to or from the
 prompt or a file.  The simplest functions for data Input and Output
 (I/O) are easy to use, but only provide limited control of how
-data is processed.  For more control, a set of functions modelled
+data is processed.  For more control, a set of functions modeled
 after the C standard library are also provided by Octave.
 
 @menu
 * Basic Input and Output::      
 * C-Style I/O Functions::       
 @end menu
 
 @node Basic Input and Output
@@ -383,17 +383,17 @@ the line @samp{Free Software is needed f
 @DOCSTRING(fskipl)
 
 @node Formatted Output
 @subsection Formatted Output
 
 This section describes how to call @code{printf} and related functions.
 
 The following functions are available for formatted output.  They are
-modelled after the C language functions of the same name, but they
+modeled after the C language functions of the same name, but they
 interpret the format template differently in order to improve the
 performance of printing vector and matrix values.
 
 @DOCSTRING(printf)
 
 @DOCSTRING(fprintf)
 
 @DOCSTRING(sprintf)
diff --git a/doc/interpreter/preface.txi b/doc/interpreter/preface.txi
--- a/doc/interpreter/preface.txi
+++ b/doc/interpreter/preface.txi
@@ -82,17 +82,17 @@ various other ways (listed alphabeticall
 
 @ifclear AUTHORSONLY
 Special thanks to the following people and organizations for
 supporting the development of Octave:
 
 @itemize @bullet
 @item
 The United States Department of Energy, through grant number
-DE-FG02-04ER25635.
+@nospell{DE-FG02-04ER25635}.
 
 @item
 Ashok Krishnamurthy, David Hudak, Juan Carlos Chaves, and Stanley
 C. Ahalt of the Ohio Supercomputer Center.
 
 @item
 The National Science Foundation, through grant numbers CTS-0105360,
 CTS-9708497, CTS-9311420, CTS-8957123, and CNS-0540147.
@@ -128,17 +128,17 @@ Academic Development Funds grant.
 @item
 The State of Texas, for providing funding through the Texas
 Advanced Technology Program under Grant No. 003658-078.
 
 @item
 Noel Bell, Senior Engineer, Texaco Chemical Company, Austin Texas.
 
 @item
-John A. Turner, Group Leader, Continuum Dynamics (CCS-2), Los Alamos
+John A. Turner, Group Leader, Continuum Dynamics @nospell{(CCS-2)}, Los Alamos
 National Laboratory, for registering the @url{octave.org} domain name.
 
 @item
 James B. Rawlings, Professor, University of Wisconsin-Madison,
 Department of Chemical and Biological Engineering.
 
 @item
 Richard Stallman, for writing GNU.
diff --git a/doc/interpreter/sparse.txi b/doc/interpreter/sparse.txi
--- a/doc/interpreter/sparse.txi
+++ b/doc/interpreter/sparse.txi
@@ -386,17 +386,17 @@ matrix.  @xref{fig:spmatrix}, for an exa
 @center @image{spmatrix,4in}
 @caption{Structure of simple sparse matrix.}
 @end float
 
 One use of sparse matrices is in graph theory, where the
 interconnections between nodes are represented as an adjacency
 matrix.  That is, if the i-th node in a graph is connected to the j-th
 node.  Then the ij-th node (and in the case of undirected graphs the
-ji-th node) of the sparse adjacency matrix is non-zero.  If each node
+@nospell{ji-th} node) of the sparse adjacency matrix is non-zero.  If each node
 is then associated with a set of coordinates, then the @dfn{gplot}
 command can be used to graphically display the interconnections
 between nodes.
 
 As a trivial example of the use of @dfn{gplot} consider the example,
 
 @example
 @group
@@ -698,17 +698,17 @@ non-zero terms which is a significant im
 
 The Cholesky@tie{}factorization itself can be used to determine the
 appropriate sparsity preserving reordering of the matrix during the
 factorization, In that case this might be obtained with three return
 arguments as @code{[r, p, q] = chol (A); spy (r)}.
 
 @float Figure,fig:simplechol
 @center @image{spchol,4in}
-@caption{Structure of the un-permuted Cholesky@tie{}factorization of the above matrix.}
+@caption{Structure of the unpermuted Cholesky@tie{}factorization of the above matrix.}
 @end float
 
 @float Figure,fig:simplecholperm
 @center @image{spcholperm,4in}
 @caption{Structure of the permuted Cholesky@tie{}factorization of the above matrix.}
 @end float
 
 In the case of an asymmetric matrix, the appropriate sparsity
diff --git a/doc/interpreter/stmt.txi b/doc/interpreter/stmt.txi
--- a/doc/interpreter/stmt.txi
+++ b/doc/interpreter/stmt.txi
@@ -765,17 +765,17 @@ endfor
 @end example
 
 @node The unwind_protect Statement
 @section The unwind_protect Statement
 @cindex @code{unwind_protect} statement
 @cindex @code{unwind_protect_cleanup}
 @cindex @code{end_unwind_protect}
 
-Octave supports a limited form of exception handling modelled after the
+Octave supports a limited form of exception handling modeled after the
 unwind-protect form of Lisp.  
 
 The general form of an @code{unwind_protect} block looks like this:
 
 @example
 @group
 unwind_protect
   @var{body}
diff --git a/doc/interpreter/strings.txi b/doc/interpreter/strings.txi
--- a/doc/interpreter/strings.txi
+++ b/doc/interpreter/strings.txi
@@ -91,17 +91,17 @@ Represents a literal backslash, @samp{\}
 
 @item \"
 Represents a literal double-quote character, @samp{"}.
 
 @item \'
 Represents a literal single-quote character, @samp{'}.
 
 @item \0
-Represents the ``nul'' character, control-@@, ASCII code 0.
+Represents the null character, control-@@, ASCII code 0.
 
 @item \a
 Represents the ``alert'' character, control-g, ASCII code 7.
 
 @item \b
 Represents a backspace, control-h, ASCII code 8.
 
 @item \f
@@ -153,18 +153,18 @@ by using @code{is_dq_string} and @code{i
 @DOCSTRING(is_dq_string)
 
 @DOCSTRING(is_sq_string)
 
 @node Character Arrays
 @section Character Arrays
 
 The string representation used by Octave is an array of characters, so
-internally the string "dddddddddd" is actually a row vector of length 10
-containing the value 100 in all places (100 is the ASCII code of "d").  This
+internally the string @nospell{"dddddddddd"} is actually a row vector of length
+10 containing the value 100 in all places (100 is the ASCII code of "d").  This
 lends itself to the obvious generalization to character matrices.  Using a
 matrix of characters, it is possible to represent a collection of same-length
 strings in one variable.  The convention used in Octave is that each row in a
 character matrix is a separate string, but letting each column represent a
 string is equally possible.
 
 The easiest way to create a character matrix is to put several strings
 together into a matrix.
diff --git a/libinterp/corefcn/qz.cc b/libinterp/corefcn/qz.cc
--- a/libinterp/corefcn/qz.cc
+++ b/libinterp/corefcn/qz.cc
@@ -337,17 +337,17 @@ with @var{Q} and @var{Z} orthogonal (uni
 As in form [2], but allows ordering of generalized eigenpairs\n\
 for (e.g.) solution of discrete time algebraic Riccati equations.\n\
 Form 3 is not available for complex matrices, and does not compute\n\
 the generalized eigenvectors @var{V}, @var{W}, nor the orthogonal matrix\n\
 @var{Q}.\n\
 \n\
 @table @var\n\
 @item opt\n\
-for ordering eigenvalues of the GEP pencil.  The leading block\n\
+for ordering eigenvalues of the @nospell{GEP} pencil.  The leading block\n\
 of the revised pencil contains all eigenvalues that satisfy:\n\
 \n\
 @table @asis\n\
 @item \"N\"\n\
 = unordered (default)\n\
 \n\
 @item \"S\"\n\
 = small: leading block has all |lambda| @leq{} 1\n\
diff --git a/libinterp/dldfcn/ccolamd.cc b/libinterp/dldfcn/ccolamd.cc
--- a/libinterp/dldfcn/ccolamd.cc
+++ b/libinterp/dldfcn/ccolamd.cc
@@ -130,20 +130,20 @@ unsorted or contains duplicate entries, 
 @code{@var{stats}(6)} is the last seen duplicate or out-of-order row\n\
 index in the column index given by @code{@var{stats}(5)}, or zero if no\n\
 such row index exists.  @code{@var{stats}(7)} is the number of duplicate\n\
 or out-of-order row indices.  @code{@var{stats}(8 : 20)} is always zero in\n\
 the current version of @sc{ccolamd} (reserved for future use).\n\
 \n\
 The authors of the code itself are S. Larimore, T. Davis (Univ. of Florida)\n\
 and S. Rajamanickam in collaboration with J. Bilbert and E. Ng.  Supported\n\
-by the National Science Foundation (DMS-9504974, DMS-9803599, CCR-0203270),\n\
-and a grant from Sandia National Lab.  See\n\
-@url{http://www.cise.ufl.edu/research/sparse} for ccolamd, csymamd, amd,\n\
-colamd, symamd, and other related orderings.\n\
+by the National Science Foundation\n\
+@nospell{(DMS-9504974, DMS-9803599, CCR-0203270)}, and a grant from Sandia\n\
+National Lab.  See @url{http://www.cise.ufl.edu/research/sparse} for\n\
+ccolamd, csymamd, amd, colamd, symamd, and other related orderings.\n\
 @seealso{colamd, csymamd}\n\
 @end deftypefn")
 {
   octave_value_list retval;
 
 #ifdef HAVE_CCOLAMD
 
   int nargin = args.length ();
@@ -385,20 +385,20 @@ unsorted or contains duplicate entries, 
 @code{@var{stats}(6)} is the last seen duplicate or out-of-order row\n\
 index in the column index given by @code{@var{stats}(5)}, or zero if no\n\
 such row index exists.  @code{@var{stats}(7)} is the number of duplicate\n\
 or out-of-order row indices.  @code{@var{stats}(8:20)} is always zero in\n\
 the current version of @sc{ccolamd} (reserved for future use).\n\
 \n\
 The authors of the code itself are S. Larimore, T. Davis (Uni of Florida)\n\
 and S. Rajamanickam in collaboration with J. Bilbert and E. Ng.  Supported\n\
-by the National Science Foundation (DMS-9504974, DMS-9803599, CCR-0203270),\n\
-and a grant from Sandia National Lab.  See\n\
-@url{http://www.cise.ufl.edu/research/sparse} for ccolamd, csymamd, amd,\n\
-colamd, symamd, and other related orderings.\n\
+by the National Science Foundation\n\
+@nospell{(DMS-9504974, DMS-9803599, CCR-0203270)}, and a grant from Sandia\n\
+National Lab.  See @url{http://www.cise.ufl.edu/research/sparse} for\n\
+ccolamd, csymamd, amd, colamd, symamd, and other related orderings.\n\
 @seealso{symamd, ccolamd}\n\
 @end deftypefn")
 {
   octave_value_list retval;
 
 #if HAVE_CCOLAMD
 
   int nargin = args.length ();
diff --git a/libinterp/dldfcn/fftw.cc b/libinterp/dldfcn/fftw.cc
--- a/libinterp/dldfcn/fftw.cc
+++ b/libinterp/dldfcn/fftw.cc
@@ -51,17 +51,17 @@ the @code{fftw} function can be used to 
 @example\n\
 @var{wisdom} = fftw (\"dwisdom\")\n\
 @end example\n\
 \n\
 @noindent\n\
 will save the existing wisdom used by Octave to the string @var{wisdom}.\n\
 This string can then be saved to a file and restored using the @code{save}\n\
 and @code{load} commands respectively.  This existing wisdom can be\n\
-reimported as follows\n\
+re-imported as follows\n\
 \n\
 @example\n\
 fftw (\"dwisdom\", @var{wisdom})\n\
 @end example\n\
 \n\
 If @var{wisdom} is an empty string, then the wisdom used is cleared.\n\
 \n\
 During the calculation of Fourier transforms further wisdom is generated.\n\
diff --git a/libinterp/interpfcn/graphics.cc b/libinterp/interpfcn/graphics.cc
--- a/libinterp/interpfcn/graphics.cc
+++ b/libinterp/interpfcn/graphics.cc
@@ -9920,17 +9920,17 @@ arguments might be given, depending on t
 \n\
 The supported property types are:\n\
 \n\
 @table @code\n\
 @item string\n\
 A string property.  @var{arg} contains the default string value.\n\
 \n\
 @item any\n\
-An un-typed property.  This kind of property can hold any octave\n\
+An @nospell{un-typed} property.  This kind of property can hold any octave\n\
 value.  @var{args} contains the default value.\n\
 \n\
 @item radio\n\
 A string property with a limited set of accepted values.  The first\n\
 argument must be a string with all accepted values separated by\n\
 a vertical bar ('|').  The default value can be marked by enclosing\n\
 it with a '@{' '@}' pair.  The default value may also be given as\n\
 an optional second string argument.\n\
diff --git a/scripts/general/structfun.m b/scripts/general/structfun.m
--- a/scripts/general/structfun.m
+++ b/scripts/general/structfun.m
@@ -59,18 +59,18 @@
 ## call in case @var{func} generates an error.  The form of the function is
 ##
 ## @example
 ## function [@dots{}] = errfunc (@var{se}, @dots{})
 ## @end example
 ##
 ## @noindent
 ## where there is an additional input argument to @var{errfunc} relative to
-## @var{func}, given by @var{se}.  This is a structure with the elements
-## "identifier", "message" and "index", giving respectively the error
+## @var{func}, given by @nospell{@var{se}}.  This is a structure with the
+## elements "identifier", "message" and "index", giving respectively the error
 ## identifier, the error message, and the index into the input arguments
 ## of the element that caused the error.  For an example on how to use
 ## an error handler, @pxref{docXcellfun, @code{cellfun}}.
 ##
 ## @seealso{cellfun, arrayfun, spfun}
 ## @end deftypefn
 
 function varargout = structfun (func, S, varargin);
diff --git a/scripts/image/ind2rgb.m b/scripts/image/ind2rgb.m
--- a/scripts/image/ind2rgb.m
+++ b/scripts/image/ind2rgb.m
@@ -21,22 +21,23 @@
 ## @deftypefnx {Function File} {[@var{R}, @var{G}, @var{B}] =} ind2rgb (@var{x}, @var{map})
 ## Convert an indexed image to red, green, and blue color components.
 ##
 ## The image @var{x} must be an indexed image which will be converted using the
 ## colormap @var{map}.  If @var{map} does not contain enough colors for the
 ## image, pixels in @var{x} outside the range are mapped to the last color in
 ## the map.
 ##
-## The output may be a single RGB image (MxNx3 matrix where M and N are the
-## original image @var{x} dimensions, one for each of the red, green and blue
-## channels).  Alternatively, the individual red, green, and blue color matrices
-## of size MxN may be returned.
+## The output may be a single RGB image (@nospell{MxNx3} matrix where M and N
+## are the original image @var{x} dimensions, one for each of the red, green
+## and blue channels).  Alternatively, the individual red, green, and blue
+## color matrices of size @nospell{MxN} may be returned.
 ##
-## Multi-dimensional indexed images (of size MxNx1xK) are also supported.
+## Multi-dimensional indexed images (of size @nospell{MxNx1xK}) are also
+## supported.
 ##
 ## @seealso{rgb2ind, ind2gray, hsv2rgb, ntsc2rgb}
 ## @end deftypefn
 
 ## Author: Tony Richardson <arichard@stark.cc.oh.us>
 ## Created: July 1994
 ## Adapted-By: jwe
 
diff --git a/scripts/image/rgb2ind.m b/scripts/image/rgb2ind.m
--- a/scripts/image/rgb2ind.m
+++ b/scripts/image/rgb2ind.m
@@ -31,17 +31,18 @@
 ## @deftypefnx {Function File} {[@var{x}, @var{map}] =} rgb2ind (@var{R}, @var{G}, @var{B}, @var{n})
 ## @deftypefnx {Function File} {[@var{x}, @var{map}] =} rgb2ind (@var{R}, @var{G}, @var{B}, @var{tol})
 ## @deftypefnx {Function File} {[@var{x}, @var{map}] =} rgb2ind (@var{R}, @var{G}, @var{B}, @var{map}, @var{dither_option})
 ## @deftypefnx {Function File} {[@var{x}, @var{map}] =} rgb2ind (@var{R}, @var{G}, @var{B}, @var{n}, @var{dither_option})
 ## @deftypefnx {Function File} {[@var{x}, @var{map}] =} rgb2ind (@var{R}, @var{G}, @var{B}, @var{tol}, @var{dither_option})
 ## Convert an image in red-green-blue (RGB) color space to an indexed image.
 ##
 ## The input image @var{rgb} must be an N-dimensional RGB image
-## (MxNxO@dots{}x3 array) where M,N,O@dots{} are the image dimensions, and the
+## (@nospell{MxNxO}@dots{}x3 array) where M,N,O@dots{} are the image
+## dimensions, and the
 ## final dimension contains the values in the red, green and blue
 ## channels.  Alternatively, the red, green and blue color channels can
 ## be input as separate arrays @var{R}, @var{G} and  @var{B}.
 ##
 ## The input @var{map} defines the colormap to be used.  Alternatively,
 ## @var{n} or @var{tol} may be used to define the maximum number of
 ## colors to use in an automatically generated colormap.  @var{n} is
 ## related to @var{tol} by:  @var{n} = (floor (1/@var{tol}) + 1)^3;
diff --git a/scripts/io/importdata.m b/scripts/io/importdata.m
--- a/scripts/io/importdata.m
+++ b/scripts/io/importdata.m
@@ -38,28 +38,28 @@
 ##
 ## @item @var{header_rows}
 ## Number of header rows before the data begins.  (Only valid for ascii files)
 ## @end itemize
 ##
 ## Different file types are supported:
 ##
 ## @itemize
-## @item Ascii table
+## @item ASCII table
 ##
-## Importing ascii table using the specified number of header rows and
+## Importing ASCII table using the specified number of header rows and
 ## the specified delimiter.
 ##
 ## @item Image file
 ##
 ## @item @sc{matlab} file
 ##
 ## @item Spreadsheet files (depending on external software)
 ##
-## @item Wav file
+## @item WAV file
 ##
 ## @end itemize
 ##
 ## @seealso{textscan, dlmread, csvread, load}
 ## @end deftypefn
 
 ## Author: Erik Kjellson <erikiiofph7@users.sourceforge.net>
 
diff --git a/scripts/linear-algebra/duplication_matrix.m b/scripts/linear-algebra/duplication_matrix.m
--- a/scripts/linear-algebra/duplication_matrix.m
+++ b/scripts/linear-algebra/duplication_matrix.m
@@ -18,31 +18,31 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} duplication_matrix (@var{n})
 ## Return the duplication matrix
 ## @tex
 ##  $D_n$
 ## @end tex
 ## @ifnottex
-##  @math{Dn}
+##  @nospell{@math{Dn}}
 ## @end ifnottex
 ##  which is the unique
 ## @tex
 ##  $n^2 \times n(n+1)/2$
 ## @end tex
 ## @ifnottex
 ##  @math{n^2} by @math{n*(n+1)/2}
 ## @end ifnottex
 ##  matrix such that
 ## @tex
 ##  $D_n * {\rm vech} (A) = {\rm vec} (A)$
 ## @end tex
 ## @ifnottex
-##  @math{Dn vech (A) = vec (A)}
+##  @nospell{@math{Dn vech (A) = vec (A)}}
 ## @end ifnottex
 ##  for all symmetric
 ## @tex
 ##  $n \times n$
 ## @end tex
 ## @ifnottex
 ##  @math{n} by @math{n}
 ## @end ifnottex
diff --git a/scripts/linear-algebra/housh.m b/scripts/linear-algebra/housh.m
--- a/scripts/linear-algebra/housh.m
+++ b/scripts/linear-algebra/housh.m
@@ -42,17 +42,17 @@
 ## threshold for zero  (usually should be the number 0)
 ## @end table
 ##
 ## @noindent
 ## Outputs (see Golub and Van Loan):
 ##
 ## @table @var
 ## @item beta
-## If beta = 0, then no reflection need be applied (zer set to 0)
+## If beta = 0, then no reflection need be applied (@nospell{zer} set to 0)
 ##
 ## @item housv
 ## householder vector
 ## @end table
 ## @end deftypefn
 
 ## Author: A. S. Hodel
 ## Created: August 1995
diff --git a/scripts/miscellaneous/mkoctfile.m b/scripts/miscellaneous/mkoctfile.m
--- a/scripts/miscellaneous/mkoctfile.m
+++ b/scripts/miscellaneous/mkoctfile.m
@@ -48,23 +48,23 @@
 ##
 ## @item -M
 ## @itemx --depend
 ## Generate dependency files (.d) for C and C++ source files.
 ##
 ## @item -R DIR
 ## Add the run-time path to the link command.
 ##
-## @item -Wl,@dots{}
-## Pass flags though the linker like "-Wl,-rpath=@dots{}".
+## @item @nospell{-Wl,@dots{}}
+## Pass flags though the linker like @nospell{"-Wl,-rpath=@dots{}"}.
 ## The quotes are needed since commas are interpreted as command
 ## separators.
 ##
 ## @item -W@dots{}
-## Pass flags though the compiler like "-Wa,OPTION".
+## Pass flags though the compiler like @nospell{"-Wa,OPTION"}.
 ##
 ## @item -c
 ## Compile but do not link.
 ##
 ## @item -g
 ## Enable debugging options for compilers.
 ##
 ## @item -o FILE
diff --git a/scripts/plot/comet3.m b/scripts/plot/comet3.m
--- a/scripts/plot/comet3.m
+++ b/scripts/plot/comet3.m
@@ -21,17 +21,17 @@
 ## @deftypefnx {Function File} {} comet3 (@var{x}, @var{y}, @var{z})
 ## @deftypefnx {Function File} {} comet3 (@var{x}, @var{y}, @var{z}, @var{p})
 ## @deftypefnx {Function File} {} comet3 (@var{hax}, @dots{})
 ## Produce a simple comet style animation along the trajectory provided by
 ## the input coordinate vectors (@var{x}, @var{y}, @var{z}).  If only @var{z}
 ## is specified then @var{x}, @var{y} default to the indices of @var{z}.
 ##
 ## The speed of the comet may be controlled by @var{p}, which represents the
-## time each point is displayed before moving tothe next one.  The default for
+## time each point is displayed before moving to the next one.  The default for
 ## @var{p} is 0.1 seconds.
 ##
 ## If @var{hax} is specified the animation is produced in that axis rather than
 ## the current axis.
 ## @seealso{comet}
 ## @end deftypefn
 
 ## Author: jwe
diff --git a/scripts/plot/gco.m b/scripts/plot/gco.m
--- a/scripts/plot/gco.m
+++ b/scripts/plot/gco.m
@@ -22,17 +22,17 @@
 ## Return a handle to the current object of the current figure, or a handle
 ## to the current object of the figure with handle @var{fig}.  The current
 ## object of a figure is the object that was last clicked on.  It is stored
 ## in the CurrentObject property of the target figure.
 ##
 ## If the last mouse click didn't occur on any child object of the figure,
 ## the current object is the figure itself.
 ##
-## If no mouse click occured in the target figure, this function returns an
+## If no mouse click occurred in the target figure, this function returns an
 ## empty matrix.
 ##
 ## Note that the value returned by this function is not necessarily the same
 ## as the one returned by gcbo during callback execution.  An executing
 ## callback can be interrupted by another callback and the current object
 ## can be modified.
 ##
 ##@seealso{gcbo, gcf, gca, get, set}
diff --git a/scripts/plot/legend.m b/scripts/plot/legend.m
--- a/scripts/plot/legend.m
+++ b/scripts/plot/legend.m
@@ -120,17 +120,17 @@
 ##
 ## @item labels
 ##   A cell array of strings of the labels in the legend.
 ## @end table 
 ##
 ## The legend label text is either provided in the call to @code{legend} or
 ## is taken from the DisplayName property of graphics objects.  If no
 ## labels or DisplayNames are available, then the label text is simply
-## "data1", "data2", @dots{}, "dataN".
+## "data1", "data2", @dots{}, @nospell{"dataN"}.
 ## @end deftypefn
 
 function [hlegend2, hobjects2, hplot2, text_strings2] = legend (varargin)
 
   if (nargin > 0
       && (! ishandle (varargin{1})
           || (strcmp (get (varargin{1}, "type"), "axes")
               && ! strcmp (get (varargin{1}, "tag"), "legend"))))
diff --git a/scripts/plot/print.m b/scripts/plot/print.m
--- a/scripts/plot/print.m
+++ b/scripts/plot/print.m
@@ -170,17 +170,17 @@
 ## or if there is no filename it is sent to the printer as postscript.
 ##
 ## @item -d@var{ghostscript_device}
 ##   Additional devices are supported by Ghostscript.
 ## Some examples are;
 ##
 ##   @table @code
 ##   @item ljet2p
-##     HP LaserJet IIP
+##     HP LaserJet @nospell{IIP}
 ##
 ##   @item ljet3
 ##     HP LaserJet III
 ##
 ##   @item deskjet
 ##     HP DeskJet and DeskJet Plus
 ##
 ##   @item cdj550
@@ -258,17 +258,17 @@
 ## @group
 ## figure (1);
 ## clf ();
 ## surf (peaks);
 ## print -dsvg figure1.svg
 ## @end group
 ## @end example
 ##
-## Example: Print to an HP Deskjet 550C.
+## Example: Print to an HP DeskJet 550C.
 ##
 ## @example
 ## @group
 ## figure (1);
 ## clf ();
 ## surf (peaks);
 ## print -dcdj550
 ## @end group
diff --git a/scripts/signal/fftshift.m b/scripts/signal/fftshift.m
--- a/scripts/signal/fftshift.m
+++ b/scripts/signal/fftshift.m
@@ -19,25 +19,25 @@
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} fftshift (@var{x})
 ## @deftypefnx {Function File} {} fftshift (@var{x}, @var{dim})
 ## Perform a shift of the vector @var{x}, for use with the @code{fft}
 ## and @code{ifft} functions, in order the move the frequency 0 to the
 ## center of the vector or matrix.
 ##
 ## If @var{x} is a vector of @math{N} elements corresponding to @math{N}
-## time samples spaced by @math{dt}, then
+## time samples spaced by @nospell{@math{dt}}, then
 ## @code{fftshift (fft (@var{x}))} corresponds to frequencies
 ##
 ## @example
 ## f = [ -(ceil((N-1)/2):-1:1)*df 0 (1:floor((N-1)/2))*df ]
 ## @end example
 ##
 ## @noindent
-## where @nospell{@math{df}} = 1 / @math{dt}.
+## where @nospell{@math{df} = 1 / @math{dt}}.
 ##
 ## If @var{x} is a matrix, the same holds for rows and columns.  If
 ## @var{x} is an array, then the same holds along each dimension.
 ##
 ## The optional @var{dim} argument can be used to limit the dimension
 ## along which the permutation occurs.
 ## @end deftypefn
 
diff --git a/scripts/signal/periodogram.m b/scripts/signal/periodogram.m
--- a/scripts/signal/periodogram.m
+++ b/scripts/signal/periodogram.m
@@ -43,17 +43,17 @@
 ## @item nfft: number of frequency bins, default max (256, 2.^ceil (log2 (length (x)))).
 ##
 ## @item Fs: sampling rate, default 1.
 ##
 ## @item range: "@nospell{onesided}" computes spectrum from [0..nfft/2+1].
 ## "@nospell{twosided}" computes spectrum from [0..nfft-1].  These strings
 ## can appear at any position in the list input arguments after window.
 ##
-## @item Pxx: one-, or two-sided power spectrum.
+## @item @nospell{Pxx}: one-, or two-sided power spectrum.
 ##
 ## @item w: angular frequency [0..2*pi) (two-sided) or [0..pi] one-sided.
 ##
 ## @item f: frequency [0..Fs) (two-sided) or [0..Fs/2] one-sided.
 ## @end itemize
 ## @end deftypefn
 
 ## Author: FL <Friedrich.Leisch@ci.tuwien.ac.at>
diff --git a/scripts/sparse/pcg.m b/scripts/sparse/pcg.m
--- a/scripts/sparse/pcg.m
+++ b/scripts/sparse/pcg.m
@@ -201,19 +201,20 @@
 ## References:
 ##
 ## @enumerate
 ## @item
 ## C.T. Kelley, @cite{Iterative Methods for Linear and Nonlinear Equations},
 ## SIAM, 1995. (the base PCG algorithm)
 ##
 ## @item
-## Y. Saad, @cite{Iterative Methods for Sparse Linear Systems}, PWS 1996.
-## (condition number estimate from PCG) Revised version of this book is
-## available online at @url{http://www-users.cs.umn.edu/~saad/books.html}
+## Y. Saad, @cite{Iterative Methods for Sparse Linear Systems}, 
+## @nospell{PWS} 1996. (condition number estimate from PCG)
+## Revised version of this book is available online at
+## @url{http://www-users.cs.umn.edu/~saad/books.html}
 ## @end enumerate
 ##
 ## @seealso{sparse, pcr}
 ## @end deftypefn
 
 ## Author: Piotr Krzyzanowski <piotr.krzyzanowski@mimuw.edu.pl>
 ## Modified by: Vittoria Rezzonico <vittoria.rezzonico@epfl.ch>
 ##  - Add the ability to provide the pre-conditioner as two separate matrices
diff --git a/scripts/statistics/tests/f_test_regression.m b/scripts/statistics/tests/f_test_regression.m
--- a/scripts/statistics/tests/f_test_regression.m
+++ b/scripts/statistics/tests/f_test_regression.m
@@ -13,18 +13,18 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {[@var{pval}, @var{f}, @var{df_num}, @var{df_den}] =} f_test_regression (@var{y}, @var{x}, @var{rr}, @var{r})
-## Perform an F test for the null hypothesis rr * b = r in a classical
-## normal regression model y = X * b + e.
+## Perform an F test for the null hypothesis @nospell{rr * b = r} in a
+## classical normal regression model y = X * b + e.
 ##
 ## Under the null, the test statistic @var{f} follows an F distribution
 ## with @var{df_num} and @var{df_den} degrees of freedom.
 ##
 ## The p-value (1 minus the CDF of this distribution at @var{f}) is
 ## returned in @var{pval}.
 ##
 ## If not given explicitly, @var{r} = 0.
diff --git a/scripts/statistics/tests/t_test_regression.m b/scripts/statistics/tests/t_test_regression.m
--- a/scripts/statistics/tests/t_test_regression.m
+++ b/scripts/statistics/tests/t_test_regression.m
@@ -13,30 +13,33 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {[@var{pval}, @var{t}, @var{df}] =} t_test_regression (@var{y}, @var{x}, @var{rr}, @var{r}, @var{alt})
-## Perform an t test for the null hypothesis @code{@var{rr} * @var{b} =
-## @var{r}} in a classical normal regression model @code{@var{y} =
-## @var{x} * @var{b} + @var{e}}.  Under the null, the test statistic @var{t}
-## follows a @var{t} distribution with @var{df} degrees of freedom.
+## Perform a t test for the null hypothesis
+## @nospell{@code{@var{rr} * @var{b} = @var{r}}} in a classical normal
+## regression model @code{@var{y} = @var{x} * @var{b} + @var{e}}.  Under the
+## null, the test statistic @var{t} follows a @var{t} distribution with
+## @var{df} degrees of freedom.
 ##
 ## If @var{r} is omitted, a value of 0 is assumed.
 ##
 ## With the optional argument string @var{alt}, the alternative of
 ## interest can be selected.  If @var{alt} is @code{"!="} or
 ## @code{"<>"}, the null is tested against the two-sided alternative
-## @code{@var{rr} * @var{b} != @var{r}}.  If @var{alt} is @code{">"}, the
-## one-sided alternative @code{@var{rr} * @var{b} > @var{r}} is used.
-## Similarly for @var{"<"}, the one-sided alternative @code{@var{rr} *
-## @var{b} < @var{r}} is used.  The default is the two-sided case.
+## @nospell{@code{@var{rr} * @var{b} != @var{r}}}.  If @var{alt} is @code{">"},
+## the one-sided alternative
+## @nospell{@code{@var{rr} * @var{b} > @var{r}}} is used.  Similarly for
+## @var{"<"}, the one-sided alternative
+## @nospell{@code{@var{rr} * @var{b} < @var{r}}} is used.  The default is the
+## two-sided case.
 ##
 ## The p-value of the test is returned in @var{pval}.
 ##
 ## If no output argument is given, the p-value of the test is displayed.
 ## @end deftypefn
 
 ## Author: KH <Kurt.Hornik@wu-wien.ac.at>
 ## Description: Test one linear hypothesis in linear regression model
diff --git a/scripts/strings/strjoin.m b/scripts/strings/strjoin.m
--- a/scripts/strings/strjoin.m
+++ b/scripts/strings/strjoin.m
@@ -19,23 +19,23 @@
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{str} =} strjoin (@var{cstr})
 ## @deftypefnx {Function File} {@var{str} =} strjoin (@var{cstr}, @var{delimiter})
 ## Join the elements of the cell-string array, @var{cstr}, into a single
 ## string.
 ##
 ## If no @var{delimiter} is specified, the elements of @var{cstr}
-## seperated by a space.
+## separated by a space.
 ##
 ## If @var{delimiter} is specified as a string, the cell-string array is
 ## joined using the string.  Escape sequences are supported.
 ##
 ## If @var{delimiter} is a cell-string array whose length is one less
-## than @var{cstr}, then the elemennts of @var{cstr} are joined by
+## than @var{cstr}, then the elements of @var{cstr} are joined by
 ## interleaving the cell-string elements of @var{delimiter}.  Escape
 ## sequences are not supported.
 ##
 ## @example
 ## @group
 ## strjoin (@{'Octave','Scilab','Lush','Yorick'@}, '*')
 ##       @result{} 'Octave*Scilab*Lush*Yorick'
 ## @end group
diff --git a/scripts/strings/strsplit.m b/scripts/strings/strsplit.m
--- a/scripts/strings/strsplit.m
+++ b/scripts/strings/strsplit.m
@@ -17,17 +17,17 @@
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {[@var{cstr}] =} strsplit (@var{s})
 ## @deftypefnx {Function File} {[@var{cstr}] =} strsplit (@var{s}, @var{del})
 ## @deftypefnx {Function File} {[@var{cstr}] =} strsplit (@dots{}, @var{name}, @var{value})
 ## @deftypefnx {Function File} {[@var{cstr}, @var{matches}] =} strsplit (@dots{})
 ## Split the string @var{s} using the delimiters specified by @var{del}
-## and return a cell-string array of sub-strings.  If a delmiter is not
+## and return a cell-string array of sub-strings.  If a delimiter is not
 ## specified the string, @var{s}, is split at whitespace.  The delimiter,
 ## @var{del} may be a string, a scalar cell-string, or cell-string array.
 ## @var{del} must be a cell-string array.  By default, consecutive
 ## delimiters in the input string, @var{s}, are collapsed into one.
 ##
 ## The second output, @var{matches}, returns the delmiters which were matched
 ## in the original string.
 ##
